docs/ocis/development/beta-testplan.md

```diff
@@ -0,0 +1,293 @@
+| Share a file by public link.                                                                                       | Link is created and can be accessed.                                   |         |
+| Share a folder by public link.                                                                                     | Link is created and can be accessed.                                   |         |
+| Share a file with another user.                                                                                    | It is shared correctly.                                                |         |
+| Share a folder with another user.                                                                                  | It is shared correctly.                                                |         |
+| Share a file with a group.                                                                                         | It is shared correctly.                                                |         |
+| Share a folder with a group.                                                                                       | It is shared correctly.                                                |         |
+| Share a folder with userB giving edit permissions. As userB do CRUD operations on items inside the received folder | userB doesn't find any problem while interacting with files.           |         |
+| Use your mobile device to access the UI                                                                            | All elements reachable                                                 |         |
+| do tests mentioned in the [spaces](#spaces) section using the web-UI                                               |                                                                        |         |
+
+## Desktop Client
+
+Prerequisite:
+- [connect ocis to your preferred LDAP server](https://owncloud.dev/ocis/deployment/ocis_ldap/)
+- create users and groups in LDAP
+- use your preferred OS for the desktop client
+
+| Test Case                                                                                               | Expected Result                                                              | Comment |
+|---------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|---------|
+| Set up two clients with the same user. Change files, add some, delete some, move some, create folders.  | Changes sync properly in both clients without errors.                        |         |
+| Share a file using contextual menu with userB.                                                          | Option to share appears in the contextual menu and file is correctly shared. |         |
+
+
+## Mobile Clients (iOS || Android)
```

Comment:
maybe also give to users link-instruction how configure the mobile client. Eventually the user would be able to download something on his computer and see it on his smartphone.